core/domain/prod_validation_jobs_one_off.py

```diff
@@ -4878,6 +4878,35 @@ def _get_custom_validation_functions(cls):
             cls._validate_collections_are_marked_deleted]
 
 
+class PseudonymizedUserModelValidator(BaseUserModelValidator):
+    """Class for validating PseudonymizedUserModels."""
+
+    @classmethod
+    def _get_model_id_regex(cls, unused_item):
+        return r'^pid_[a-z]{32}$'
+
+    @classmethod
+    def _get_external_id_relationships(cls, item):
+        return {}
+
+    @classmethod
+    def _validate_user_settings_with_same_id_not_exist(cls, item):
+        """Validates that the UserSettingsModel with the same ID as this model
```

Comment:
I'm a bit confused about this. Would there ever have been such a model, given that these models start with uid_?